scss/treeview/_layout.scss

```diff
@@ -21,6 +21,16 @@
             position: relative;
         }
 
+        // Wrappers
+        .k-top,
+        .k-mid,
+        .k-bot {
+            display: flex;
+            flex-direction: row;
+            align-items: center;
+            align-content: center;
```

Comment:
It seems like the layout will change for the icons and checkboxes, when the item content wraps (has `white-space: normal`). Previously the icons and checkboxes were aligned with the item top, now they will be centered. Not sure if that is intentional.